kentikapi/internal/testutil/http.go

```diff
@@ -34,22 +35,26 @@ func NewSpyHTTPHandler(t testing.TB, responseCode int, responseBody []byte) *Spy
 }
 
 func (h *SpyHTTPHandler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
-	h.RequestsCount++
-	h.LastMethod = r.Method
-	h.LastURL = r.URL
-	h.LastHeader = r.Header
-
-	body, err := ioutil.ReadAll(r.Body)
-	assert.NoError(h.t, err)
-	h.LastRequestBody = string(body)
-
-	err = r.Body.Close()
-	assert.NoError(h.t, err)
-
-	rw.Header().Set("Content-Type", "application/json")
-	rw.WriteHeader(h.responseCode)
-	_, err = rw.Write(h.responseBody)
-	assert.NoError(h.t, err)
+	if r.Method == "PRI" {
+		log.Printf("Omitting HTTP/2 connection preface (PRI)\n")
```

Comment:
```suggestion
		h.t.log.Printf("SpyHTTPHandler: omitting HTTP/2 connection preface (PRI)")
```
IMO it's better to use logger provided by testing package than importing new one. We use that style in all other tests.
Also, it's better to add context which part of testing code is logging. It gets confusing quickly when you don't remember the code exactly.
BTW: we found that those new lines are not needed

The same for MultipleResponseSpyHTTPHandler

Reply:
Done